src/main/scripts/org/reaktivity/specification/nukleus/oauth/control/resolve/one.realm.with.issuer.and.audience/controller.rpt

```diff
@@ -31,8 +31,10 @@ write ${controlR05C.nextCorrelationId()}
 write ${core:string("oauth")}
 write [0x05] "RS256"
 write 0
-write [0x0B] "test issuer"      # issuer extension
-write [0x0D] "test audience"    # audience extension
+write ${oauth:resolveEx()
+              .issuer("test issuer")
+              .audience("test audience")
+              .build()}
```

Comment:
Each `.` should be vertically aligned with the `:` for readability.
Please apply that approach to all scripts.